### ISSUE TITLE
Mechanism to cleanup uploaded files when error occurs

### ISSUE THREAD

**Issue by xabarass** [balderdashy/skipper#125]
Hi, lets say i want to add limit on upload file size. Skipper provides option `maxBytes` for that and it throws an error when defined size is exceeded, just as expected.
Consider an use case when uploading several files, but file limit is exceeded on i.e. second file, first file is completely saved on disk and error is thrown. Error object doesn't provide any information about already saved files that can be cleaned up or manipulated in any way. This can lead to potentially storing huge amount of data on server without any information of them presented to application.
One workaround is to manually do cleanup of old and unused files, but that is not a pretty solution.
Is there an more elegant way to handle this scenario? Maybe limiting uploads to single file?

**Comment by sailsbot** [comment]
Thanks for posting, @xabarass.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
